data/ai/micro_ais/scenarios/bottleneck_defense.cfg

```diff
@@ -90,13 +90,13 @@
         [message]
             speaker=Big Bad Orc
             image=$profile~FL()~RIGHT()
-            message= _ "They there!  We them get!"
+            message= _ "They there! We them get!"
```

Comment:
`# po:` this is Orcish speech, deliberately omitting the "are" part of "they're"

Reply:
Agree, but that's outside the scope of this PR.

Reply:
93df584adb2e5a6e5c9a9de1c579b81d1bbddee8